resources/views/PageDesign/Partials/Head.twig

```diff
@@ -78,8 +79,10 @@
 {% set performanceLevel = config('Ceres.log.performance.level') %}
 {% set itemName = config('Ceres.item.name') %}
 
+<link rel="preload" as="style" href="{{ defaultCSS }}">
 {% if LayoutContainer.show("Ceres::Template.StyleOverwrite") | trim is empty %}
-    <link data-sass-root="{{ plugin_path('Ceres') }}" data-sass-original="{{ plugin_path('Ceres') }}/css/ceres-legacy.scss" rel="stylesheet" href="{{ plugin_path('Ceres') }}/css/ceres-legacy.css">
+    <link data-sass-root="{{ plugin_path('Ceres')
```

Comment:
Bitte in einer Zeile

Reply:
Ups, das war noch von einem anderen Test